COM-2157:new landing pages for shopper notification

Rameshdesaiise opened the pull request:
Created 3 new templates for shopper notifications.

michaelsadlonkibo commented on labels/en-US.json:
```diff
@@ -601,6 +601,14 @@
+  "shopperIntransitConfirmationSMS": "Customer {0} is on there way to pickup order number {1}",
+  "shopperInStoreConfirmation": "<p>Thank you for letting us know you are here.</p>",
+  "shopperItemsQty": "Qty:",
+  "shoperNotificationIAmHere": "I'm Already Here",
```
`shoperNotificationIAmHere` should be `shopper` with 2 p's.

michaelsadlonkibo commented on labels/en-US.json:
```diff
@@ -601,6 +601,14 @@
   "shippingTo": "Items shipping to",
   "shipTo": "Ship To",
   "shortDesc": "Summary",
+  "shopperAtStorenotified": "Store has already been notified.",
```
`shopperAtStorenotified` should be `shopperAtStoreNotified`

michaelsadlonkibo commented on labels/en-US.json:
```diff
@@ -601,6 +601,14 @@
   "shortDesc": "Summary",
+  "shopperAtStorenotified": "Store has already been notified.",
+  "shopperClosingNotes": "Thank you for shopping with Retailer {0}",
+  "shopperIntransitConfirmationSMS": "Customer {0} is on there way to pickup order number {1}",
```
`shopperIntransitConfirmationSMS` should be `shopperInTransitConfirmationSms`

michaelsadlonkibo commented on labels/en-US.json:
```diff
@@ -601,6 +601,14 @@
+  "shopperItemsQty": "Qty:",
+  "shoperNotificationIAmHere": "I'm Already Here",
+  "shopperNotificationText": "<p>Your order is available and ready for you to pick up. Please click the \"I'm on my way\" button to inform the store you are on your way to pick up your order.</p> <p>Thank you so much for your purchase! We're happy we can help you to get what you're looking for.</p> <br/><br/>",
+  "shopperOnMyWayConfirmationText": "<p>Thank you for confirming. The store has been notified, Please contact the store for additional communication.</p>",
```
`The store has been notified, Please contact the store for additional communication.`
should be
`The store has been notified. Please contact the store for additional communication.`
That comma should be a period.

michaelsadlonkibo commented on stylesheets/back-office.css:
```diff
@@ -542,4 +547,99 @@ table {
\ No newline at end of file
+}
+
+.Shopper-shipment-item-info {
```
Minor but ideally this would be lowercase `s`.

michaelsadlonkibo reviewed (changes requested):
Looks good just some minor text or variable naming issues.

Commit:
COM-2157 Added 3 New Landing Pages for Shopper Notification

COM-2157:new landing pages for shopper notification

Created 3 new templates for shopper notifications.

COM-2157 Added Flag for SentCustomerAtStoreNotification

COM-2157 Fixed the confirmation Flag

atulchavanKibo committed:
COM-2352: Blocking customer pickupinfo submit button to restrict multiple clicks

Commit:
COM-2157 FIxed Flag
